src/PageLayout/PageLayout.test.tsx

```diff
@@ -123,7 +123,9 @@ describe('PageLayout', () => {
       <ThemeProvider>
         <PageLayout>
           <PageLayout.Header aria-label="header">Header</PageLayout.Header>
-          <PageLayout.Content aria-label="content">Content</PageLayout.Content>
+          <PageLayout.Content as="main" aria-label="content">
```

Comment:
Instead of adding the `as` prop to PageLayout.Content, I'd prefer to nest the <main> element inside the content:

```suggestion
          <PageLayout.Content>
            <main aria-label="content>
```

The `as` prop API is often too permissive and creates additional type complexity. Can you think of any downsides to nesting the <main> tag inside the content component?

Reply:
I don't think it really matters in the context of this test. I think this will require a change to the assertions though.